core/src/main/java/edu/wpi/grip/core/operations/networktables/NTPublishOperation.java

```diff
@@ -28,9 +28,12 @@
     private final List<Method> ntValueMethods = new ArrayList<>();
 
     public NTPublishOperation(Class<T> type) {
-        this.table = NetworkTable.getTable("GRIP");
         this.type = checkNotNull(type, "Type was null");
 
+        // NetworkTables automatically initializes itself when we get a table.  We don't want this behavior.
+        table = NetworkTable.getTable("GRIP");
+        NetworkTable.shutdown();
```

Comment:
This is an "invalid fix"?  Did we even have that label before, or did you create it especially for this PR?

Anyway, the problem was that NetworkTables was logging messages even when disabled.  You personally complained about this a lot.  This change fixes that issue.  If you have opinions about changes we could possibly make to the user interface, feel free to open an issue for them.